deutsche-gesellschaft-fur-psychologie.csl

```diff
@@ -279,6 +283,65 @@
       <text variable="locator" prefix=" "/>
     </group>
   </macro>
+  <macro name="issued-citation">
+    <choose>
+      <if variable="issued">
+        <group>
+          <choose>
+            <if variable="original-date">
+              <date variable="original-date">
+                <date-part name="year"/>
+              </date>
+              <choose>
+                <if variable="translator">
+                  <text value="; übers. "/>
+                </if>
+                <else>
+                  <text value="/"/>
+                </else>
+              </choose>
+            </if>
+          </choose>
+          <text macro="issued-year"/>
+        </group>
+      </if>
+      <else>
+        <group delimiter="-">
+          <text term="no date" form="short"/>
+          <text variable="year-suffix"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="original-date">
+    <date variable="original-date">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="publication-history">
+    <group prefix="(" suffix=")">
+      <choose>
+        <if variable="references">
+          <text variable="references"/>
+        </if>
+        <else>
+          <choose>
+            <if variable="original-date">
+              <group>
+                <text value="Original erschienen"/>
```

Comment:
Use the "originally-published" term instead

Reply:
From what I just learned, it is the term "orignal-work-published". Will add this.

Reply:
Apparently, if I use the term "originally-pubished", the validator will not accept the CSL. When I use "original-work-published", the Validator is happy. However, importing it to Zotero, it gives me an error, sayin the CSL is not valid. What I got from the debug log is this:
`failed to validate: stdin:35:6: error: attribute ^name with invalid value "original-work-published" stdin:324:16: error: attribute ^term with invalid value "original-work-published" error: invalid input `

I am riddled. It does actually work and does what it's supposed to do, but gives me an error when installing the style.
What am I missing?

Reply:
Alright, so I checked and I can't find a single .csl in the entire repository that uses "originally-published" nor "original-work-published" (the latter would be correct, corresponding to the CSL 1.0.2 specs). However, Zotero does not like it, so I assume this is a bug within Zotero's CSL validator.

I created a report with Zotero (https://forums.zotero.org/discussion/94053/report-id-1661039569#latest).

My question is: should I implement the CSL as it should be (using the term) or would I leave it with the hard coded text element so the CSL could be used, assuming the changes might be merged at some point?

Reply:
Zotero has not implemented CSL 1.0.2 yet and the term was introduces with this version. @bwiernik, would it be acceptable to avoid using the term and go with the hard coded text and update this later, when Zotero has implemented CSL 1.0.2?